components/central-application-gateway/README.md

```diff
@@ -110,6 +110,12 @@ As a result, Central Application Gateway:
 A combination of `{API_BUNDLE_NAME}` and `{API_DEFINITION_NAME}` which are extracted from an Application CR must be unique for a given application.
 Invocation of endpoints with duplicate names results in a `400 Bad Request` failure. In such a case, you must change one of the names to avoid ambiguity.
 
+### Status codes for errors returned by Application Gateway
+
+- when application specified in path doesn't exist Application Gateway returns `404 Not Found`
+- when application, service or entry for Compass use case is not specified in the path, Application Gateway returns `400 Bad Request `
+- when call to target API timeouts Application Gateway returns `504 Gateway Timeout`
```

Comment:
```suggestion
- when call to target API timeouts, Application Gateway returns `504 Gateway Timeout`
```

Reply:
```suggestion
- When a call to the target API times out, Application Gateway returns `504 Gateway Timeout`.
```
- `Timeout` is a noun ("what") 
- `(to) time out` is a phrasal verb, where the verb (`time`) part is conjugated, and the preposition part remains the same (`time out` -> `times out` / `timed out`; ~`timeouts`~, unless you mean the noun in plural/~`timeouted`~).